aten/src/ATen/core/ivalue.cpp

```diff
@@ -39,6 +39,11 @@ TORCH_API c10::intrusive_ptr<ConstantString> ConstantString::create(
   return c10::make_intrusive<ConstantString>(std::move(str_));
 }
 
+TORCH_API c10::intrusive_ptr<ConstantString> ConstantString::create(
+    c10::string_view str_) {
+  return c10::make_intrusive<ConstantString>(std::string(str_));
```

Comment:
NB: this is OK because ConstantString actually does store a std::string internally (wao)